google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/SpannerOptions.java

```diff
@@ -17,6 +17,7 @@
 package com.google.cloud.spanner;
 
 import com.google.api.core.ApiFunction;
+import com.google.api.core.InternalApi;
```

Comment:
Looks like a bad merge here?

Reply:
Yep, the change to ServiceOptions had not been merged in. I'll add it.

Reply:
The change to ServiceOptions has been added.